src/components/example.js

```diff
@@ -19,22 +20,21 @@ import { __ } from '@wordpress/i18n'
  * @param {ExampleProps} props
  * @return {React.ReactElement} An example preview component.
  */
-const Example = ({ attributes }) => (
-  <div>
-    <p>
-      {__(
-        'This is an alternate preview component',
-        'headless-blocks'
-      )}
-    </p>
-    <p>
-      {__(
-        'Here is the value of the new-field attribute, if it exists:',
-        'headless-blocks'
-      )}
-    </p>
-    <p>{attributes['new-field']}</p>
-  </div>
-)
+const Example = (props) => {
+  const attributes = props.attrs ? props.attrs : props.attributes;
```

Comment:
On the front-end (with https://github.com/johnstonphilip/headless-blocks/pull1)), the attributes will be in `props.attr`.

In the block editor, the attributes will be in `props.attributes`